_component_design/date-picker.md

```diff
@@ -0,0 +1,341 @@
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-specifications.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Button states
+Text and icons should always be centered vertically and horizontally.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption:"Date picker button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-button-states-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption:"Month picker button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-button-states-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Date picker spacing
+The date picker element appears below corresponding text input that controls it. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-spacing.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+## Date selection interactions
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Single date selection
+ * Keyboard tabbing onto the input field and/or clicking on it automatically open the date picker
+ * The date may be filled in by hand, according to the format expressed in the placeholder
+ * When the user must pick one date, the picker should have one field showing expected date format in the placeholder
+ * Each date in the picker is hoverable, but only one date may be selected
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-single-date.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Date ranges using two fields
+The traditional method for date range selection is to use two separate fields, with a picker for each field.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/date-picker/date-picker-date-ranges-using-two-fields.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Picking the start date
```

Comment:
Not sure if we have a plugin that handles H4 level headings. If there is one include it here, otherwise we can leave this for the time being. Or perhaps use .hxSubSectionTitle. Your call.